services/federator/src/Federator/API.hs

```diff
@@ -57,13 +76,15 @@ data API route = API
 -- TODO: the client ids in the 'PrekeyBundle' aren't really needed here.  do we want to make a
 -- new type for that, then?
 
+-- TODO: rename
 data FUser = FUser
   { _fuGlobalHandle :: !(Qualified Handle),
     _fuFQU :: !(Qualified UserId)
   }
   deriving (Eq, Show, Generic)
 
-deriveJSON (wireJsonOptions "_fu") ''FUser
+-- TODO: use Generics
```

Comment:
TODO comments that won't be tackled in the PR they are introduced - don't we usually call those FUTUREWORK? Maybe rename?

Reply:
Yes, you're right about the conventions. I'd still like to fix it before merging this PR.

Reply:
All resolved now.